moviesurfer/cli/main.py

```diff
@@ -1,2 +1,19 @@
+import os
+import wget
+import zipfile
+
+# Function to download data
+def download_data():
+    output_dir = './moviesurfer/data'
```

Comment:
It may be better to add this as an argument. Inside an actual CLI function we can add this one as a default with an opportunity to override

Reply:
👍